### PR TITLE
Transformations Update

### PR DESCRIPTION
**Pull request by thkwznk**

### CODE REVIEW

**Review by martincapello — CHANGES_REQUESTED** [review]
Looking great! I just made a minor suggestion and requested to transform the commented line into an issue.

File: rotate.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+
+## Rotation Pivot
+
+![Rotation Pivot](rotate/pivot-point-context-bar.png)
+
+Selection is rotated around a single, defined point (![Rotation Pivot](rotate/pivot-point.png)). By default the pivot point is set in the center of the selection and is not visible until you start rotating the image.
+
+Its placement and visibility settings can be changed from the context bar of any selection tool. Rotation pivot can also be moved with a mouse by holding <kbd>Left click</kbd> and dragging the point.
+
+<!-- FUTURE: Expand tools section with selection tools and link them here -->
```
[the remark this answers is not in the document]
Comment (thkwznk) [reply rotate.md:13]: Done 👍

File: rotate.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+## Rotation Algorithms
+
+![Rotation Algorithms](rotate/rotation-algorithms.png)
+
+There are two rotation algorithms available:
+
+- Fast Rotation
+- [RotSprite](https://en.wikipedia.org/wiki/Pixel-art_scaling_algorithms#RotSprite)
+
+While RotSprite is widely accepted as an algorithm producing better results, the choice depends only on users' preferences.
```
[the remark this answers is not in the document]
Comment (thkwznk) [reply rotate.md]: Changed to "the choice is up to your preferences."

**Review by martincapello — APPROVED** [review]
Looks good to me! 🥇

**Pull request merged**